### PR TITLE
Door open animation

### PR DESCRIPTION
**Pull request by arejayelle**
Doors open 
+ Small cleanup of branch 2 where doors weren't associated w/ their respective passage
https://user-images.githubusercontent.com/35614138/143974229-5e574cc3-a91c-4b79-80de-0c9e25760160.mp4

### CODE REVIEW

**Review by daniela-venuta — DISMISSED** [review]

File: Hush/Assets/Scripts/Doors/Door.cs [hunk 1]
```diff
@@ -57,10 +63,12 @@ private void OpenDoor(InputAction.CallbackContext context)
             if (!CanOpenDoor)
                 return;
 
-            // TODO Door opening animation
-            wall.SetActive(false);
-            
-            // respawn back into scene
+            anim.SetTrigger("Open");
+            Invoke("LoadNext", 2f);
```
Comment (po-trottier) [inline Hush/Assets/Scripts/Doors/Door.cs:67]:
```suggestion
            Invoke(nameof(LoadNext), 2f);
```

File: Hush/Assets/Scripts/Doors/Door.cs [hunk 2]
```diff
@@ -57,10 +63,12 @@ private void OpenDoor(InputAction.CallbackContext context)
             if (!CanOpenDoor)
                 return;
 
-            // TODO Door opening animation
-            wall.SetActive(false);
-            
-            // respawn back into scene
+            anim.SetTrigger("Open");
```
Comment (po-trottier) [inline Hush/Assets/Scripts/Doors/Door.cs:66]: Use Static Getter here

**Review by emilie-martin — DISMISSED** [review]
looks clean, love the anim! ❤️ 👍

File: Hush/Assets/Scripts/Doors/Door.cs (hunk 1 above)
Comment (arejayelle) [reply Hush/Assets/Scripts/Doors/Door.cs:67]: Changed in 9454f5848f7592ea000731146dfcc48c658a318a

File: Hush/Assets/Scripts/Doors/Door.cs (hunk 2 above)
Comment (arejayelle) [reply Hush/Assets/Scripts/Doors/Door.cs:66]: 85b1fabd16c39da2be518f24f4da3d341236f871

**Review by daniela-venuta — APPROVED** [review]

**Review by emilie-martin — DISMISSED** [review]

File: Hush/Assets/Scripts/Doors/Door.cs (hunk 2 above)
Comment (arejayelle) [reply Hush/Assets/Scripts/Doors/Door.cs]: Use Class 252db7ee5e211be92895448519ea25fb682ca5d3

**Review by po-trottier — APPROVED** [review]

**Pull request merged**